src/Model/Command/UserRecommendation.php

```diff
@@ -29,10 +29,12 @@ class UserRecommendation extends AbstractCommand implements UserAwareInterface
     private $hardRotation = false;
     /** @var string */
     private $minimalRelevance = self::MINIMAL_RELEVANCE_LOW;
-    /** @var array */
+    /** @var string[] */
     private $filters = ['valid_to >= NOW'];
     /** @var string|null */
     private $modelName = null;
+    /** @var string[] */
+    private $properties = null; // @v2.0 use ['item_id'] as default
```

Comment:
If `['item_id'] ` will be default and required in v2.0, should you be able to reset it (so also remove the item_id and possibly trigger the old format) using `setProperties()`? I think you shouldn't... So maybe add the `['item_id']` always, and use this array only for additional properties?